### PR TITLE
Distributed Data Parallel Example

### PR DESCRIPTION
**Pull request by andreyvelich**
Example without workflows

Fixes https://github.com/kubeflow/pytorch-operator/issues/29

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pytorch-operator/73)

### CODE REVIEW

File: examples/mnist-ddp/cpu/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    pip install -v .
+
+RUN /opt/conda/bin/conda config --set ssl_verify False
+RUN pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org
+RUN pip install --trusted-host pypi.org --trusted-host files.pythonhosted.org torchvision
+WORKDIR /workspace
+RUN chmod -R a+w /workspace
+
+ADD . /opt/pytorch_dist_mnist
+ENTRYPOINT ["mpirun", "-n", "4", "--allow-run-as-root", "python", "-u", "/opt/pytorch_dist_mnist/mnist_DDP_CPU.py"]
```
Comment (johnugeorge) [inline examples/mnist-ddp/cpu/Dockerfile]: can you add newline at end of each file ?

Comment (andreyvelich) [reply examples/mnist-ddp/cpu/Dockerfile]: Done

**Pull request merged**